src/pages/CommandView/CommandView.tsx

```diff
@@ -53,7 +53,7 @@ const CommandView = () => {
     const build = formBuilder(system, command)
     description = command.description
     formElement = (
-      <Box>
+      <>
```

Comment:
I don't see a reason not to move all of this directly into the `Box` below. Assigning it to a variable doesn't seem to accomplish anything.